tests/test_invenio_assets_filters.py

```diff
@@ -80,11 +80,13 @@ def test_clean_CSS(self, static_dir):
         app = Flask(__name__)
         InvenioAssets(app)
         with app.app_context():
+            filter = CleanCSSFilter()
+            result = 'h1{font-family:Verdana;color:#FFF}'
```

Comment:
warning here, I think that the version 3 returns `#FFF` while the version 4 returns `#fff`...